### PR TITLE
Include section comment count to hearing comment count

### PR DESCRIPTION
**Pull request by Rikuoja**

### CODE REVIEW

**Comment by Rikuoja** [comment]
Fix #160

File: democracy/models/base.py [hunk 1]
```diff
@@ -121,9 +121,16 @@ class Commentable(models.Model):
 
     def recache_n_comments(self):
         new_n_comments = self.comments.count()
+        # if commentable has sections, include them in the total comment count
+        if hasattr(self, 'sections'):
+            for section in self.sections.all():
```
Comment (akx) [inline democracy/models/base.py]: Use Django's `Sum` aggregate? (Or at the very least, use `.values_list()` to query only the `n_comments` field you need.

**Comment by akx** [comment]
Missing tests.

**Comment by akx** [comment]
CI failing still.

Please fold in the test changes to the commit making the corresponding changes in the actual code; otherwise using `git bisect` will be painful.

**Pull request merged**